plugins/woocommerce/client/legacy/css/admin.scss

```diff
@@ -8798,4 +8803,4 @@ html:has(#status-table-templates){
 // Fix for Safari bug: https://bugs.webkit.org/show_bug.cgi?id=280063.
 .woocommerce-admin-page #postbox-container-2 {
 	clear: left;
-}
\ No newline at end of file
+}
```

Comment:
Was this added automatically? If not, let's remove it.

Reply:
It was!